crypto/fipsmodule/ec/p384.c

```diff
@@ -581,7 +581,7 @@ static void fiat_p384_mul_scalar_rwnaf(int16_t *out, const unsigned char *in) {
 }
 
 // fiat_p384_select_point selects the |idx|-th projective point from the given
-// precomputed table and copies it to |out| in constant time.
+// precomputed table and copies it to |out| in constant-time.
```

Comment:
I think they hyphen is not needed here. It would have been needed if we say "in constant-time fashion"